src/cppcache/src/MapSegment.hpp

```diff
@@ -41,6 +40,9 @@
 #include <ace/Versioned_Namespace.h>
 #include "TombstoneList.hpp"
 #include <unordered_map>
+
+#include "util/concurrent/spinlock_mutex.hpp"
```

Comment:
`"` includes are for "local" includes. If `spinlock_mutex` was an external library then `<` would be used.

If "down here" means why the sub directories, then the answer is to be consistent with the namespace. The utility of that is debatable.